packages/bits/demo/src/components/demo/dialog/dialog-inside-overlay/dialog-inside-overlay.example.component.html

```diff
@@ -12,11 +12,12 @@
     <nui-dialog-header title="Dialog Title" i18n-title (closed)="close()"></nui-dialog-header>
     <div class="dialog-body">
         <p>
-            <nui-select-v2 placeholder="Select an item" i18n-placeholder>
+            <nui-select-v2 class="mb-5" placeholder="Select an item" i18n-placeholder>
                 <nui-select-v2-option *ngFor="let option of options2" [value]="option">
                     {{ option }}
                 </nui-select-v2-option>
             </nui-select-v2>
+            <nui-date-time-picker class="date-time-picker" (modelChanged)="onModelChanged($event)" displayMode="inline" [model]="dt" [appendToBody]="true"></nui-date-time-picker>
```

Comment:
We should avoid adding things to our examples that are just for testing purposes. If we need a component for testing, do you mind creating a separate one? That way we can avoid cluttering up the example.

Reply:
Yeah, I agree. Created a separate one for testing purposes, thanks